src/Compilers/CSharp/Portable/CodeGen/EmitArrayInitializer.cs

```diff
@@ -372,37 +372,61 @@ private static bool IsMultidimensionalInitializer(ImmutableArray<BoundExpression
 
 #nullable enable
 
-        private bool TryEmitReadonlySpanAsBlobWrapper(NamedTypeSymbol spanType, BoundExpression wrappedExpression, bool used, bool inPlace, BoundExpression? start = null, BoundExpression? length = null)
+        private bool TryEmitReadonlySpanAsBlobWrapper(NamedTypeSymbol spanType, BoundExpression wrappedExpression, bool used, BoundExpression inPlaceTarget, out bool avoidInPlace, BoundExpression? start = null, BoundExpression? length = null)
```

Comment:
>BoundExpression inPlaceTarget, out bool avoidInPlace [](http://example.com/codeflow?start=126&length=52)

I think we can benefit from detailed documentation for these parameters. How they are supposed to be used, etc.

Reply:
Done